src/coredump/coredump.c

```diff
@@ -1248,6 +1255,7 @@ static int process_kernel(int argc, char* argv[]) {
         context[CONTEXT_SIGNAL]    = argv[1 + CONTEXT_SIGNAL];
         context[CONTEXT_TIMESTAMP] = argv[1 + CONTEXT_TIMESTAMP];
         context[CONTEXT_RLIMIT]    = argv[1 + CONTEXT_RLIMIT];
+        context[CONTEXT_HOSTNAME]  = argv[1 + CONTEXT_HOSTNAME];
```

Comment:
iovec[] needs to be enlarged by one here, too, no?

Reply:
Hmm, I believe the size got automatically increased because I have added a new entry to the CONTEXT_* enum.

Have a look at the variable definition:
```
char* context[_CONTEXT_MAX] = {};
```

Have I forgotten how C works?

Reply:
iovec[] is not context[]…

Reply:
Yay. True!